doom-modeline.el

```diff
@@ -234,6 +234,7 @@ If DEFAULT is non-nil, set the default mode-line for all buffers."
         (add-hook 'dashboard-mode-hook #'doom-modeline-set-project-modeline)
         (add-hook 'image-mode-hook #'doom-modeline-set-media-modeline)
         (add-hook 'magit-mode-hook #'doom-modeline-set-vcs-modeline)
+        (add-hook 'magit-status-mode-hook #'doom-modeline-set-vcs-modeline)
```

Comment:
@seanfarley Sorry to comment in the closed PR. I don't understand why adding the hook for `magit-status-mode` but not others like `magit-diff-mode`, etc. Since they are derived from `magit-mode`, I don't think it's necessary.

Reply:
No worries :-)

I would have thought so too but for me `(magit-status)` didn't have the modeline until I added the hooks. As for `magit-diff-mode`: currently there is no modeline that appears for me. I think you should add those hooks too!

Reply:
It's wired. In my env, for `magit-mode`, `magit-diff-mode`, `magit-log-mode`, etc. , there are `doom-modeline-vcs`. Actually, if the modeline is not set for the buffer, the `doom-modeline-main` will be applied. So it's wired if there is no modeline, unless you have some special configurations.

Reply:
Ah, I think this may from `doom`. It seems magit is customized there:

https://github.com/hlissner/doom-emacs/blob/ceb17e0e290c264f691dfe1386d50e9c0ee10ed9/modules/ui/modeline/config.el#L52

Reply:
Great! I've revert the change to remove the hook.